.github/workflows/docker.yml

```diff
@@ -0,0 +1,166 @@
+name: docker
+on:
+  push:
+  # TODO: Build on main?
```

Comment:
The question here is, basically, should we build / push on merge? Right?

My first thought is: sure, why not? As long as we aren't bumping the `latest` tag.

Reply:
yes, this builds on pushes and `main`. But it I'd think it might be better to build on `tags` and `main` only, and not on on `push` because generally pushes are unstable, and can cause a lot of unnecessary CI jobs.